.github/workflows/vnu-jar-problem-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "vnu-jar",
+      "pattern": [
+        {
+          "regexp": "^\"file:(.*)\":(\\d+).(\\d+)-\\d+\\.\\d+:\\s(?:info )(error|warning):\\s(.*)$",
+          "file": 1,
+          "line": 2,
+          "column": 3,
```

Comment:
As validation is run on generated HTML files, line numbers and file name will end up marking wrong positions. I think we can fix the filename part by renaming the file to have same name as source (for validation only; revert it once validation step is done), but we should skip the line/column numbers as they'll be misleading.

Reply:
I think running it on the original file will just give the wrong lines, since there are all the boilerplate from Bikeshed/respec that would make it not line up.
I guess this current version would mostly help those that do have the generated files in the source, but wouldn't change anything for those that don't

Reply:
Sorry, I wasn't clear. I mean, we should rename the built file to the one in repo (source file) so GitHub can show annotations on the right file, even if the line numbers are wrong.